<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pexpay</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="pexpayApiKey_lineEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="binanceApiKey_lineEdit"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="huobiApiKey_lineEdit"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Bybit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="pexpaySecretKey_lineEdit"/>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLineEdit" name="bybitSecretKey_lineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Binance</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="savePath_lineEdit"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Huobi</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="binanceSecretKey_lineEdit"/>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1" alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="QPushButton" name="saveSettings_pushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLineEdit" name="bybitApiKey_lineEdit"/>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Secret key</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Secret key</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Secret key</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Secret key</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="huobiSecretKey_lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Save path</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
